contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_string_integers_increasing.py

```diff
@@ -173,27 +138,10 @@ def _validate(
         execution_engine: ExecutionEngine = None,
     ) -> Dict:
 
-        SIMI = metrics.get("column_values.string_integers.monotonically_increasing")
+        SIMI = metrics.get("column_values.string_integers.increasing")
```

Comment:
minor nit, could you make the [variable name lowercase](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) and spell it out a bit more (it's ok if it's really long).

Reply:
done